src/geometry/manifold/charted_space.lean

```diff
@@ -797,29 +797,56 @@ variables (e : local_homeomorph α H)
 space `α`, then that local homeomorphism induces an `H`-charted space structure on `α`.
 (This condition is equivalent to `e` being an open embedding of `α` into `H`; see
 `local_homeomorph.to_open_embedding` and `open_embedding.to_local_homeomorph`.) -/
-def singleton_charted_space (h : e.source = set.univ) : charted_space H α :=
+def local_homeomorph.singleton_charted_space (h : e.source = set.univ) : charted_space H α :=
```

Comment:
Is there a reason this definition is outside the `local_homeomorph` namespace you open just after?